daemons/lvmdbusd/utils.py

```diff
@@ -681,3 +682,16 @@ def _remove_objects(dbus_objects_rm):
 # Remove dbus objects from main thread
 def mt_remove_dbus_objects(objs):
 	MThreadRunner(_remove_objects, objs).done()
+
+
+# Make stream non-blocking
+def make_non_block(stream):
```

Comment:
I was trying to figure out how to do this with `subprocess`, didn't realize calling `fnctl` on `proc.stdout` would work :man_facepalming:

Reply:
Thanks for the review and the check with `libblockdev`.  @vpodzime did a nice job coming up with the select loop in the lvmshell code.

Reply:
Did I? :thinking: